README.md

```diff
@@ -7,7 +7,7 @@ Package license: Apache-2.0
 
 Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/eccodes-feedstock/blob/master/LICENSE.txt)
 
-Summary: ECMWF ecCodes Copyright 2005-2018 ECMWF.
+Summary: ECMWF ecCodes Copyright 2005- ECMWF.
```

Comment:
The readme is auto-generated. This change should be made in the meta.yaml file and then re-render to propagate it here.

Reply:
OK. I will do another PR. Sorry

Reply:
You can use this one if you want. Just make the same modification in the meta.yaml and [then re-render](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks).